metrics/cli/dependencies.yaml

```
@@ -1,33 +1,33 @@
 # Provides an ability to build the Flutter web application.
-#
-# Consider the following link to install: https://flutter.dev/docs/get-started/install
-flutter: 1.25.0-8.2.pre
+flutter:
+  recommended_version: 1.25.0-8.2.pre
+  install_url: https://flutter.dev/docs/get-started/install
 
 # Provides an ability to create and manage Google Cloud resources.
 # Used to create and configure a GCloud project.
-#
-# Consider the following link to install: https://cloud.google.com/sdk/docs/install
-gcloud: 341.0.0
+gcloud:
+  recommended_version: 341.0.0
+  install_url: https://cloud.google.com/sdk/docs/install
 
 # Provides an ability to adapt packages of code for applications, download standalone tools, etc.
 # Used to install npm dependencies.
-#
-# Consider the following link to install: https://www.npmjs.com/get-npm
-npm: 6.14.13
+npm:
+  recommended_version: 6.14.13
+  install_url: https://www.npmjs.com/get-npm
```

:label:
It seems like the https://www.npmjs.com/get-npm link redirects to the `getting started` page.
```suggestion
  install_url: https://docs.npmjs.com/downloading-and-installing-node-js-and-npm
```